cba_settings_userconfig/cba_settings.sqf

```diff
@@ -150,14 +156,14 @@ force force ace_maptools_rotateModifierKey = 1;
 // ACE Medical
 force force ace_medical_ai_enabledFor = 2;
 force force ace_medical_AIDamageThreshold = 1;
-force force ace_medical_allowDeadBodyMovement = true;

```

Comment:
Any reason why this was removed? What's the default value?

Reply:
As far as I can tell this is no longer a valid configuration. I've loaded into a mission and exported all the current ACE3 settings after importing the server settings on top of them. That line no longer was shown in the export which leads me to believe its a remnant in our current server file that no longer does anything.